src/Sylius/Bundle/AdminBundle/Resources/views/Shipment/Show/_header.html.twig

```diff
@@ -1,6 +1,13 @@
 <h1 class="ui header">
     <i class="circular truck icon"></i>
     <div class="content">
-        {{ 'sylius.ui.shipment'|trans ~ ' for ' ~ 'sylius.ui.order'|trans ~ ' #' ~ shipment.order.number}}
+        {{ 'sylius.ui.shipment'|trans ~ ' : ' ~ 'sylius.ui.order'|trans ~ ' #' ~ shipment.order.number}}
```

Comment:
78fc1d5

Reply:
👍 for the key name but we also need the actual message for it in English over [here](https://github.com/Sylius/Sylius/blob/1.12/src/Sylius/Bundle/AdminBundle/Resources/translations/messages.en.yml).
Only English translation is necessarry since we use it as the baseline, and all other locales should be managed via crowdin.